app/components/field_container_component.rb

```diff
@@ -3,7 +3,7 @@
 class FieldContainerComponent < ViewComponent::Base
 
   renders_one :label
-  def initialize(label: nil, value: nil, truncate: false)
```

Comment:
why removing his ? 
and if you remove it, you need to remove it corresponding code and it usage (if used)

Reply:
I removed it because we are not using it in our component's code, it just appears as a argument.
and yes I removed it's usages.